Bugsnag/include/Bugsnag/BugsnagConfiguration.h

```diff
@@ -149,6 +149,18 @@ typedef BOOL (^BugsnagOnSessionBlock)(BugsnagSession *_Nonnull session);
  */
 @property(readwrite, retain, nullable) NSSet<id> *redactedKeys;
 
+/**
+ * A set of strings and NSRegularExpression objects that determines which errors should
+ * be discarded based on their `errorClass`.
+ *
+ * OnError / OnSendError blocks will not be called for discarded errors.
+ *
+ * Some examples of errorClass are: Objective-C exception names like "NSRangeException",
+ * signal names like "SIGABRT", mach exception names like "EXC_BREAKPOINT", and Swift
+ * error names like "Fatal error".
+ */
+@property(readwrite, copy, nullable) NSSet<id> *discardClasses;
```

Comment:
Should probably mention that comparisons are case sensitive, since people will invariably get this wrong.